server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/component/BranchLoader.java

```diff
@@ -27,7 +27,7 @@
 import static org.apache.commons.lang.StringUtils.trimToNull;
 
 public class BranchLoader {
-
+  public static final String DEFAULT_MAIN_BRANCH_NAME = "master";
```

Comment:
I think this constant would deserve to be in BranchDto

Reply:
ok